agent/config/config_windows.go

```diff
@@ -142,8 +142,8 @@ func DefaultConfig() Config {
 		SharedVolumeMatchFullConfig:         BooleanDefaultFalse{Value: ExplicitlyDisabled}, //only requiring shared volumes to match on name, which is default docker behavior
 		PollMetrics:                         BooleanDefaultFalse{Value: NotSet},
 		PollingMetricsWaitDuration:          DefaultPollingMetricsWaitDuration,
-		GMSACapable:                         true,
-		FSxWindowsFileServerCapable:         true,
+		GMSACapable:                         BooleanDefaultFalse{Value: ExplicitlyDisabled},
+		FSxWindowsFileServerCapable:         BooleanDefaultFalse{Value: ExplicitlyDisabled},
```

Comment:
Here we are changing the default FSsWindowsCapability from true to false, but I don't see where it's being enabled.  This is a behavior change, (as FSx Windows has been available for 2 years).  Can you point out where this is being enabled?

Reply:
I found the following initialization code for FSxWindowsFileServerCapable, can we confirm that this is still applicable? 

https://github.com/aws/amazon-ecs-agent/blob/2dcc42bd8a6f1f253537ea820b8920c9a805a7c1/agent/config/parse_windows.go#L44

Reply:
yes the area that Yash has linked is the exact area where we determine whether to advertise the FXs and gMSA capabilities.